lib/symmetric-key.js

```diff
@@ -39,7 +40,11 @@ exports.generateKey = async ({keyId, type}) => {
   };
   try {
     await database.collections.ssm.insertOne(record, database.writeOptions);
-    return {'@context': _constants.SECURITY_CONTEXT_V2_URL, id: keyId, type};
+    return {
+      '@context': WEBKMS_CONTEXT_URL,
+      id: keyId,
+      type
```

Comment:
I suspect this doesn't work with the `type` value not being defined in the above context. We'll need to include the suite context as well.

Reply:
@dlongley np, I added the suite context to it also. And added tests checking for it, for both key types.

Reply:
Thanks!